cluster/gce/coreos/configure-node.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+
+# Copyright 2015 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+function configure-hostname() {
+    hostnamectl set-hostname $(hostname | cut -f1 -d.)
+}
+
+function configure-kubelet() {
+    mkdir -p /etc/kubernetes/manifests
```

Comment:
I don't think you need this here. This path is also being created in configure_static_pods()